### PR TITLE
Move multi sht example to file

### PR DESCRIPTION
**Pull request by abrauchli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,92 +38,11 @@ from the sensor, but return the values read last. To read a new sample, make
 sure to call `readSample()`
 
 ### Sample code
```
Comment (JHahn96) [inline README.md]: Maybe we can refactor this to sample implementations, as there is no code shown anymore?

Comment (abrauchli) [reply README.md]: the code is just one link down, see the rendered version here: https://github.com/Sensirion/arduino-sht/blob/0c4b3319655d9165446dfd1566cabef09878e79f/README.md

Comment (abrauchli) [reply README.md]: Ah, like this? https://github.com/Sensirion/arduino-sht/blob/c9ef8b3a9197879c8f63b9edf3ed364e96d67a3f/README.md

**Comment by winkj** [comment]
Looks good to me! Please go ahead and merge

**Review by JHahn96 — APPROVED** [review]

**Comment by abrauchli** [comment]
@winkj thanks for the review, I've tagged 1.1.1 on it

https://github.com/Sensirion/arduino-sht/releases/tag/1.1.1

**Comment by winkj** [comment]
@abrauchli great. We should update library.properties too to make it appear in the Arduino IDE Library Manager; I won't get to it today, but hopefully next week unless you beat me to it :)

**Pull request merged**